bigquery_etl/events_daily/query_templates/event_types_v1/init.sql

```diff
@@ -0,0 +1,172 @@
+{{ header }}
```

Comment:
The content of this `init.sql` is so similar to `query.sql` that I would prefer a single `query.sql` and we can pass in an additional parameter to the template `init: bool`. If `init` is true, we'd include the CREATE OR REPLACE statement, etc. This is the approach used in the glean_baseline query generation machinery.

Reply:
I agree that this would be ideal, but these differ in pretty substantial ways: e.g. getting the `event_index` for `init` is just a `group by`, but we need to join with the previous results in `query.sql` to drop the ones we already have.

We could probably refactor things to be more amenable to this, but let's leave that as a separate issue. Alongside that refactoring we should also expand test scope.